compiler/q-implant/src/QImplant.cpp

```diff
@@ -251,6 +271,50 @@ void QImplant::write(loco::Graph *g)
     }
   }
 
+  std::set<luci::CircleNode *> visit;
+  std::deque<luci::CircleNode *> que;
+  for (auto input_node : loco::input_nodes(g))
+  {
+    auto node = loco::must_cast<luci::CircleNode *>(input_node);
+    que.emplace_back(node);
+  }
+
+  // check whether node can share its successor node quantization parameter
+  while (!que.empty())
+  {
+    auto node = que.front();
+    que.pop_front();
+
+    // skip when node is output
+    if (node->opcode() == luci::CircleOpcode::CIRCLEOUTPUT)
+    {
+      continue;
+    }
+
+    auto quantparam = node->quantparam();
+    THROW_UNLESS(quantparam);
+
+    for (auto child : loco::succs(node))
```

Comment:
plz rename `child` to something else. graph/node has no child concept.

Reply:
I never knew about that, and renamed to `successor` as is.

Thank you.